airbyte-integrations/connectors/source-graphql/README.md

```diff
@@ -0,0 +1,127 @@
+# GraphQL Source
+
+This is the repository for the GraphQL source connector, written in Python.
+
+## Local development
+
+### Prerequisites
+**To iterate on this connector, make sure to complete this prerequisites section.**
+
+#### Minimum Python version required `= 3.7.0`
+
+#### Build & Activate Virtual Environment and install dependencies
+From this connector directory, create a virtual environment:
+```
+python -m venv .venv
+```
+
+This will generate a virtualenv for this module in `.venv/`. Make sure this venv is active in your
+development environment of choice. To activate it from the terminal, run:
+```
+source .venv/bin/activate
+pip install -r requirements.txt
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv.
+
+Note that while we are installing dependencies from `requirements.txt`, you should only edit `setup.py` for your dependencies. `requirements.txt` is
+used for editable installs (`pip install -e`) to pull in Python dependencies from the monorepo and will call `setup.py`.
+If this is mumbo jumbo to you, don't worry about it, just put your deps in `setup.py` but install using `pip install -r requirements.txt` and everything
+should work as you expect.
+
+#### Building via Gradle
+You can also build the connector in Gradle. This is typically used in CI and not needed for your development workflow.
+
+To build using Gradle, from the Airbyte repository root, run:
+```
+./gradlew :airbyte-integrations:connectors:source-graphql:build
+```
+
+#### Create credentials
+**If you are a community contributor**, follow the instructions in the [documentation](https://docs.airbyte.io/integrations/sources/http-request)
+to generate the necessary credentials. Then create a file `sample_files/config.json` conforming to the `source_graphql/spec.json` file.
+Note that any directory named `sample_files` is gitignored across the entire Airbyte repo, so there is no danger of accidentally checking in sensitive information.
+See `integration_tests/sample_config.json` for a sample config file.
+
+### Locally running the connector
+```
+python main.py spec
+python main.py check --config sample_files/config.json
+python main.py discover --config sample_files/config.json
+python main.py read --config sample_files/config.json --catalog sample_files/configured_catalog.json
+```
+
+### Locally running the connector docker image
+
+#### Build
+First, make sure you build the latest Docker image:
+```
+docker build . -t airbyte/source-graphql:dev
```

Comment:
```suggestion
docker build . -t airbyte/source-graphql:0.1.0
```

Reply:
I added the two types in the README.md because the tag dev must be used for development and the tag 0.1.0 must be used for production.